### ISSUE TITLE
About implementation details

### ISSUE THREAD

**Issue by foralliance** [Hwang64/MLKP#4]
About implementation details,there are two places in the paper to describe

1.In the section 4.1:
`   We initialize the network using an ImageNet pretrained basic model [35, 16] and the weights of layers in MLKP are initialized with the method of Xavier [13].  In the first step we freeze all layers in basic model with only training the layers of MLKP and RPN. Secondly, the whole network is trained within two stages by decreasing learning rate.`

2.In the section 4.3.1:
`   Specifically, the network is trained  for  120k  iterations  for  the  first  step  with  learning rate of 0.001. Then, whole network is fine-tuned for 50k iterations with learning rate of 0.001 and 30k iterations with learning rate of 0.0001. `

----------------------------------------------------------------------------------------------------------------------------------

But these two points seem to be completely different from the contents of solver.prototxt.

**Comment by Hwang64** [comment]
There is no doubt that the training stage contains three steps as the paper illustrates, the solver.prototxt is a general solver file and if you wand to train the network, please modify it depending on your train.prototxt.

**Comment by foralliance** [comment]
@Hwang64 
谢谢您的回复

我是不是可以理解为：这种训练方式是分段的：
第一步，freeze all layers in basic model with only training the layers of MLKP and RPN. 
当这一步训练结束得到模型之后，再进行
第二步：the whole network is trained within two stages by decreasing learning rate.
